p2p/src/msg.rs

```diff
@@ -94,6 +97,8 @@ fn max_msg_size(msg_type: Type) -> u64 {
 		Type::BanReason => 64,
 		Type::GetTransaction => 32,
 		Type::TransactionKernel => 32,
+		Type::GetKernels => 48,
+		Type::Kernels => std::u64::MAX, // TODO: Determine appropriate max size
```

Comment:
Is there a maximum number of kernels per block? If not, this could be tricky.

Reply:
There's a maximum block weight (see `consensus.rs`) which implies a maximum number of kernels if the block was filled with kernels.